src/icons/ArrowIcon.js

```diff
@@ -4,7 +4,7 @@ import { Icon } from '../components/Icon'
 
 const ArrowIcon = (props) => (
   <Icon { ...props }>
-    <g transform='translate(0, -1)'>
+    <g transform='translate(0.75, -1)'>
```

Comment:
I did it by adding the rotate in the path itself instead of in the svg element. I also don't mirror, just rotate in 90 deg increments.
Check here 
https://github.com/resmio/widget/blob/master/src/components/IconArrow.jsx